pkg/asset/cluster/tfvars.go

```diff
@@ -62,6 +66,7 @@ import (
 	"github.com/openshift/installer/pkg/types/openstack"
 	"github.com/openshift/installer/pkg/types/ovirt"
 	"github.com/openshift/installer/pkg/types/powervs"
+	powervstypes "github.com/openshift/installer/pkg/types/powervs"
```

Comment:
This is the same import as the line above. Can we consolidate them?

Reply:
It seems the double import was brought back.